ui/perfherder/graphs/GraphsContainer.jsx

```diff
@@ -478,6 +498,55 @@ class GraphsContainer extends React.Component {
                       />
                     ))}
 
+                  {highlightChangelogData && changelogData.length > 0 && (
+                    <VictoryBar
+                      key="changelog"
+                      data={changelogData.map((i) => ({
+                        x: i.date,
+                        y: zoomDomain.maxY,
+                        label: i.description,
+                      }))}
+                      style={{
+                        data: { fill: '#d19900', width: 1 },
+                      }}
+                      events={[
+                        {
+                          target: 'data',
+                          eventHandlers: {
+                            onMouseOver: () => {
+                              return [
+                                {
+                                  target: 'data',
+                                  mutation: () => ({
+                                    style: { fill: '#d19900', width: 3 },
```

Comment:
Do we really need to duplicate the `fill` for each mutation? Couldn't we update only the `width`?